interface/MultiDimFit.h

```diff
@@ -72,6 +72,9 @@ class MultiDimFit : public FitterAlgoBase {
   static std::vector<float>        specifiedVals_;
   static RooArgList                specifiedList_;
   static bool saveInactivePOI_;
+  static std::string setPhysicsModelParameterExpression_;
```

Comment:
I already added this above (L57) so should probably try and remove the duplication. From past experience there was also no need to do the range setting here - it can be done at the beginning in combine.cc as normal.